### ISSUE TITLE
Promo materials!

### ISSUE THREAD

**Issue by quat1024** [ElementalCode/elemental#50]
We should think about promotion and advertising for this, like, really early.

This doesn't mean you should change your Scratch signature RIGHT NOW RIGHT NOW to something Elemental-branded - but it does mean we should at least think about what we're gonna do :)

We'll need:
- [ ] **(In progress)** a vibrant and memorable landing page
- [ ] a pitch
- [x] **(In progress)** some scratch signature images of various sizes (full-sig, square icon, wide-but-thin bar?)
- [x] "made with Elemental" optional banner people can include on their pages
- [ ] possibly a small advertising budget

I, for one, will post to /r/web_design when we're done (maybe pay for a Reddit ad, if I have any money) and of course I'll add it to my signature :smiley: 

Thoughts?

**Comment by matthewr6** [comment]
Maybe /r/internetisbeautiful?

**Comment by PullJosh** [comment]
An assortment of thoughts:  
1) **I call making sigs! :D**
2) If we end up doing ads, I'd say use Google Adwords.
 3)We should "partner" with gwiddle - on our homepage we add a shoutout to gwiddle; on gwiddle's homepage they shoutout to us. Once the project reaches a beta stage, I'll talk to jbayfield.
 4) **I've been thinking through homepage designs & pitches. I'll do a quick start on one tomorrow.**

(Bolded the things that I call doing. :P)

**Comment by matthewr6** [comment]
Sounds cool
Though I refuse to host on Gwiddle because PHP :P

**Comment by quat1024** [comment]
Talking to Gwiddle sounds greattttt!!!!!! That's nice

Maybe we could be their website builder, prepare a "gwiddle website
builder" with a "save to my gwiddle account" or smtn and they can add it to
their website.

**Comment by matthewr6** [comment]
That'd be interesting - jbayfield would have to make an API for zip file uploads.

**Comment by PullJosh** [comment]
Not integrated into theirs - we want all the traffic. :P

That's another thing, though, that I had thought of before I even made the forum topic about having this idea. I tend to plan things out. xD

**Comment by matthewr6** [comment]
Yeah we're hosting on Heroku remember :P
Also, not sure how Gwiddle works but we can link each project to a site or something

**Comment by PullJosh** [comment]
**We aren't integrating with gwiddle.** The point of partnering with them would be to give each other traffic - not to take traffic away from each other.

All we need is a link at the bottom that says "Want to host your websites online? Check out [large-scale gwiddle logo here]". Same on gwiddle, but for us.

**Comment by PullJosh** [comment]
I'm not sure what I think of this sig... any feedback?
![image](https://cloud.githubusercontent.com/assets/5458180/8892150/2b3c806e-3317-11e5-8b84-53af3d3bd4fd.png)

**Comment by matthewr6** [comment]
Ok

**Comment by PullJosh** [comment]
I'm not sure what I think of this sig. Any feedback?
![image](https://cloud.githubusercontent.com/assets/5458180/8892151/3dd7eb00-3317-11e5-912e-1a74330264d4.png)

**Comment by PullJosh** [comment]
And how about one with the fancy-pants background?
![image](https://cloud.githubusercontent.com/assets/5458180/8892152/6d0a6948-3317-11e5-971e-b69973b5e84a.png)

**Comment by quat1024** [comment]
That's the same one :P

**Comment by PullJosh** [comment]
How about this one with a fancy-pants background?
![image](https://cloud.githubusercontent.com/assets/5458180/8892156/8c3fa33c-3317-11e5-951f-8e0c0ab96a7e.png)

**Comment by quat1024** [comment]
Ooops deleted the Wring xomment

**Comment by matthewr6** [comment]
#2

**Comment by PullJosh** [comment]
@quat1024 I deleted them. That's how I try to be awesome online - I just delete my mistakes. :P

**Comment by matthewr6** [comment]
Darn you quat
Also never knew about #X referencing the Xth issue

**Comment by PullJosh** [comment]
@Firedrake969 Nice issue link. :3

**Comment by PullJosh** [comment]
So... feedback?

**Comment by quat1024** [comment]
It's not centered, and it's kinda... empty imo. I think this design would work better if "Elemental" and the subheading were on the same line and it was about a quarter as tall :stuck_out_tongue_winking_eye:

What's the font you're using btw?

Once we can get the thing looking smexy, we should add a screenshot of some blocks to fill some of the emptiness. "show, don't tell" and all that.

Re: the tagline, I'd swap "make" for "code" (cuz, idk, I think the word "code" is a bit of a turn-off? And they're also designing it), and remove either "Scratch-like" or "block based" because they kinda say the same thing. `"Learn to make websites with a simple, block based editor."`

**Comment by BookOwl** [comment]
It seems kind of small and plain. Also, it seems a little repetitive, since scratch like and block based mean about the same thing.

**Comment by matthewr6** [comment]
@PullJosh 
Background gives it more texture, which I prefer (before quat destroyed my hopes and dreams xD)
Also something seems too empty in the right of the green area

Agree with swapping make for code and removing "Scratch-like" or "block-based" (depends on our target audience - if mainly Scratch, use the former, but if not, use the latter)

**Comment by robinp7720** [comment]
@PullJosh It looks amazing, its clean and the colours defiantly fit to the app. The only problem I might is the text on the bottom is too long. It breaks the look and feel of the design

**Comment by PullJosh** [comment]
@quat1024 Fonts are source code pro and source sans pro.

**Everyone:** I agree with all of your suggestions - I wasn't too happy with it myself, but I couldn't really figure out what needed to be changed. As for the signature's size, I just set it at the maximum size a Scratch sig can be. In the future, samller versions could also be made. ;)

It's back to the drawing board for me.

**Comment by PullJosh** [comment]
Hmm... Does this look any good?
![image](https://cloud.githubusercontent.com/assets/5458180/8893662/dda8422e-3367-11e5-989c-e5cb9783bc27.png)

**Comment by BookOwl** [comment]
I would like the block with Elemental in it to be a bit bigger. Much better than the last one though

**Comment by PullJosh** [comment]
Just the block, or the text within it as well?

**Comment by an-OK-squirrel** [comment]
I like it!

**Comment by BookOwl** [comment]
The proportions should stay the same, just make them both bigger.

**Comment by PullJosh** [comment]
Okay!

**Comment by PullJosh** [comment]
@BookOwl How's this?
![image](https://cloud.githubusercontent.com/assets/5458180/8894040/5748b52e-3376-11e5-8869-2d1db6777201.png)

**Comment by matthewr6** [comment]
Something still seems empty to the left and right, but otherwise I love it (and not sure how to improve in any case).
Though we may want to change it after we decide how the blocks will look.

**Comment by BookOwl** [comment]
I like it!

**Comment by quat1024** [comment]
Looks nice! The block is a bit blurry though

**Comment by PullJosh** [comment]
@quat1024 Yeah, scaled up image. I'll make it nicer if we decide to use it.

**Comment by an-OK-squirrel** [comment]
:P

**Comment by TheInitializer** [comment]
I personally like the first one best :P

**Comment by matthewr6** [comment]
Anyone still doing this stuff?

**Comment by PullJosh** [comment]
No, but I'll keep working on it once we get closer to a decent release.

Keep open for now.

**Comment by matthewr6** [comment]
Ok, you noticed my closing spree?  :P

**Comment by PullJosh** [comment]
Just a little bit. :P

**Comment by an-OK-squirrel** [comment]
What do you guys want the made with Elemental banner to look like?

**Comment by PullJosh** [comment]
Like this:
![image](https://cloud.githubusercontent.com/assets/5458180/9039864/50427ca4-39cc-11e5-9a0c-74a276ad95af.png)

**Comment by PullJosh** [comment]
Just made an epic pun...
"With Elemental, learning to create websites is a snap!"

(We need to use that somewhere. :P)

**Comment by matthewr6** [comment]
I like both

**Comment by PullJosh** [comment]
You mean the banner and the pun?

**Comment by matthewr6** [comment]
Yeah

**Comment by an-OK-squirrel** [comment]
Thanks a lot! You just made my efforts worthless as mine would suck compared to yours xD

**Comment by PullJosh** [comment]
No problem, m8. xD

**Comment by an-OK-squirrel** [comment]
_cries_
Seriously though, looks great.

**Comment by PullJosh** [comment]
:P

**Comment by matthewr6** [comment]
Signature and org logo pls

**Comment by PullJosh** [comment]
@Firedrake969 I'm on it. :)

**Comment by PullJosh** [comment]
I tried a couple of things for the org logo, but it turned out just sort of meh.
![image](https://cloud.githubusercontent.com/assets/5458180/9316035/b6ae0f6e-4500-11e5-82eb-854b7f86cad8.png)
Any ideas on how to improve it? Or should I just try something totally different?

**Comment by matthewr6** [comment]
What's wrong?  I like it

**Comment by an-OK-squirrel** [comment]
It's cool, needs an e in the middle

**Comment by matthewr6** [comment]
> It's cool, needs an e in the middle

YES

**Comment by PullJosh** [comment]
@an-OK-squirrel I'll try that.

@Firedrake969 +1 for markdown skills

**Comment by matthewr6** [comment]
> > > > > > > > > thx

**Comment by PullJosh** [comment]
UM YES
![image](https://cloud.githubusercontent.com/assets/5458180/9317316/b1326424-4508-11e5-9766-9f5feb4a2567.png)

**Comment by matthewr6** [comment]
why does that look like internet explorer

**Comment by matthewr6** [comment]
but green

**Comment by PullJosh** [comment]
We could try a capital, but I think it would be ugly. :/

**Comment by matthewr6** [comment]
Use whatever font you used in https://cloud.githubusercontent.com/assets/5458180/8894040/5748b52e-3376-11e5-8869-2d1db6777201.png

**Comment by PullJosh** [comment]
@Firedrake969 I did. I used the font of the bottom text. I can do the top text instead if you want.

**Comment by matthewr6** [comment]
Yeah I meant the thinner one

**Comment by PullJosh** [comment]
UM NO
![image](https://cloud.githubusercontent.com/assets/5458180/9318405/525cd62e-4511-11e5-9bf2-7fc38c28239a.png)

**Comment by matthewr6** [comment]
oh

**Comment by PullJosh** [comment]
:P

**Comment by TheInitializer** [comment]
I like it without the e

**Comment by PullJosh** [comment]
@TheInitializer That's great, but you're a bit outnumbered. :P

**Comment by matthewr6** [comment]
also signature please?

**Comment by PullJosh** [comment]
Is this okay?
![image](https://cloud.githubusercontent.com/assets/5458180/9331170/df96031a-458b-11e5-86b0-34745941792f.png)

**Comment by matthewr6** [comment]
Yeah

**Comment by PullJosh** [comment]
I'll work on the other sizes when I get the chance.

**Comment by matthewr6** [comment]
TinyPic-ified for sigs:
http://oi61.tinypic.com/2h4j5ah.jpg

**Comment by PullJosh** [comment]
NOOOOOOOOOOOOOOO NOT TINYPIC

Use cubeupload.

**Comment by matthewr6** [comment]
nu

**Comment by PullJosh** [comment]
Tinypic is like torture for your eyes.
http://i.cubeupload.com/Tc4TUJ.jpg

Also, I have good news and bad news.
**Bad news:** Paint.NET crashed and I lost the .pdn file, so making different sizes of the signatues is going to be difficult (do you guys think it's worth it?)
**Good news:** I get to have pizza for lunch.

**Comment by matthewr6** [comment]
1.  nah
2.  Jimmy John's.  I win.

**Comment by an-OK-squirrel** [comment]
but does it matter?

it's not like it gives you a watermark

**Comment by PullJosh** [comment]
@an-OK-squirrel They have this thing where they like to randomly take down images for no reason and replace them with the tinypic logo.

I think it's part of their advertising campaign. :/

**Comment by an-OK-squirrel** [comment]
Really? 

<h1>Really?</h1>

**Comment by PullJosh** [comment]
Either that, or people decided it was fun to upload the tinypic logo to tinypic a bunch of times.

**Comment by an-OK-squirrel** [comment]
I could see that happening

**Comment by matthewr6** [comment]
Or they take it down themselves....
Or it's something bad that they uploaded

**Comment by an-OK-squirrel** [comment]
troll: want to wreck forums and blame it on other people.

other troll: what about tinypic?

**Comment by matthewr6** [comment]
Use whatever image host you want ok

**Comment by an-OK-squirrel** [comment]
ok m9

**Comment by matthewr6** [comment]
How's this going?

**Comment by an-OK-squirrel** [comment]
what

**Comment by PullJosh** [comment]
Are multiple sig sizes needed?

**Comment by matthewr6** [comment]
Maybe one big and one small

**Comment by PullJosh** [comment]
K. For small, should it be square or long and flat?

**Comment by matthewr6** [comment]
long and flat, since we do have a square icon thing.

**Comment by matthewr6** [comment]
Anyone want to make a sales pitch?  :P

**Comment by PullJosh** [comment]
Kind of hard to do until we finish the actual coding part. :P

**Comment by matthewr6** [comment]
@ElementalCode/programmers ok the actual coding part is done

**Comment by TheInitializer** [comment]
(mostly)

**Comment by TheInitializer** [comment]
(the _backend_ is pretty much done)

**Comment by matthewr6** [comment]
same with the core editor functionality

**Comment by TheInitializer** [comment]
pretty much but we need more blocks

**Comment by towerofnix** [comment]
many many more blocks

**Comment by matthewr6** [comment]
add them!

**Comment by PullJosh** [comment]
@Firedrake969 Says it's time for promo materials... What do I need to make?
